salvo/tools/format_python_tools.sh

```diff
@@ -28,12 +28,14 @@ EXCLUDE="--exclude=benchmarks/tmp/*,.cache/*,*/venv/*,tools/format_python_tools.
 # E124 Closing bracket does not match visual indentation
 # E125 Continuation line with same indent as next logical line
 # E126 Continuation line over-indented for hanging indent
+# W504 line break after binary operator
 
 # We ignore false positives because of what look like pytest peculiarities 
 # F401 Module imported but unused
 # F811 Redefinition of unused name from line n
-flake8 . ${EXCLUDE} --ignore=E114,E111,E501,F401,F811,E124,E125,E126,D --count --show-source --statistics
+flake8 . ${EXCLUDE} --ignore=E114,E111,E501,F401,F811,E124,E125,E126,W504,D --count --show-source --statistics
```

Comment:
Are we also planning to re-enable W504? If yes, we should add another TODO above this line mentioning it, so that we don't forget.

Reply:
No, it has conflicts with other formatting checks, we should disable it. FYI, nighthawk also disabled it, refer to https://github.com/envoyproxy/nighthawk/blob/main/tools/format_python_tools.sh#L34

Reply:
Ack, thank you for the reference.